prow/config/experimental/.defaults.yaml

```
@@ -0,0 +1,7 @@
+defaults:
+  mapping:
+    istio: istio
+  input: ./prow/cluster/jobs/
+  output: ./prow/cluster/jobs/
+  cluster: public
+  modifier: exp
```

```suggestion
  modifier: exp_ds
```

since we may have other experiments?

Or changes branches-out to `experimental-*` so we can just always have this for all experiments? I think I prefer the later.

---

Might be `.*` not `*` though, I forget the syntax here